examples/bots/abc/config.yml

```diff
@@ -33,3 +33,5 @@ rails:
   dialog:
     single_call:
       enabled: False
+
+allow_exceptions: True
```

Comment:
While reading through the implementation, it makes more sense to call this `enable_rail_exceptions` (the default value should be `False` to keep backward compatibility). Then, make all the exceptions end with `RailException` (e.g., `ActiveFenceHarassmentRailException`, `SelfCheckOutputRailException`).

Reply:
Much better indeed! I'll do it.